Description: Apply "format security" warning/error fix from Ilya Barygin. 
     I don't think it will actually cause a security issue in this context and
     i'm not sure if it will cause a FTBFS in debian (apparently it didn't back
     in 2011) but it's certainly horrible coding practice to pass a string that
     is not explicitly intended to be a format string to the first parameter of
     a printf so may  as well fix it. (Closes: #646327)
Author: Ilya Barygin <randomaction@ubuntu.com>
Reviewed-by: Peter Michael Green <plugwash@debian.org>
Bug-Debian: https://bugs.debian.org/646327

--- libbs2b-3.1.0+dfsg.orig/src/bs2bconvert.c
+++ libbs2b-3.1.0+dfsg/src/bs2bconvert.c
@@ -153,7 +153,7 @@ int main( int argc, char *argv[] )
 	if( ( infile = sf_open( infilename, SFM_READ, &sfinfo ) ) == NULL )
 	{
 		printf( "Not able to open input file %s.\n", infilename );
-		printf( sf_strerror( NULL ) );
+		printf( "%s", sf_strerror( NULL ) );
 		return 1;
 	}
 
